src/System.Text.Encoding/tests/UTF8Encoding/UTF8EncodingGetString.cs

```diff
@@ -9,17 +9,11 @@ namespace System.Text.Tests
 {
     public class UTF8EncodingGetString
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
-
         public static IEnumerable<object[]> GetString_TestData()
         {
             byte[] bytes = new byte[] { 85, 84, 70, 56, 32, 69, 110, 99, 111, 100, 105, 110, 103, 32, 69, 120, 97, 109, 112, 108, 101 };
             string expected = "UTF8 Encoding Example";
             yield return new object[] { bytes, 0, bytes.Length, expected };
-
-            int randomIndex = s_randomDataGenerator.GetInt32(-55) % bytes.Length;
-            int randomCount = s_randomDataGenerator.GetInt32(-55) % (bytes.Length - randomIndex) + 1;
-            yield return new object[] { bytes, randomIndex, randomCount, expected.Substring(randomIndex, randomCount) };
```

Comment:
Sorry if I wasn't clear.  It'd still be good to do a subset test, it just shouldn't be with random inputs, especially since this isn't really random, with the seeds hardcoded.